dev/PushNotifications/PushNotificationChannel.cpp

```diff
@@ -121,8 +121,16 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
     HRESULT __stdcall PushNotificationChannel::InvokeAll(_In_ ULONG length, _In_ byte* payload, _Out_ BOOL* foregroundHandled) noexcept try
     {
         auto args = winrt::make<winrt::Microsoft::Windows::PushNotifications::implementation::PushNotificationReceivedEventArgs>(payload, length);
-        m_foregroundHandlers(*this, args);
```

Comment:
@pmpurifoy has restructured this code. I would recommend that you rebase with his change unless you want to redo this all over again! :) 

Plaease See my email.

Reply:
I took a look into that PR. My understanding is that this being a change on API it would be preferable to get it in before Feature Complete while the PushNotification change may take a little longer.

Reply:
Ok. Please work with @pmpurifoy to rebase his change once you check this in.